source/openapi3/split.yaml

```
@@ -2,25 +2,19 @@ openapi: 3.0.0
 info:
   title: Split API
   description: >
-    Split allows you to make, get and manage payments using nothing but bank
-    accounts.
+    Split enables your business to make, collect and receive payments using nothing but bank accounts.
+    We've overlaid a simple REST API on top of the legacy (and newer) Australian payment rails, 
+    enabling all of your account to account payment flows through 1 simple integration. 
```

I usually suggest hyphenating phrasal adjectives.

```suggestion
    enabling all of your account-to-account payment flows through 1 simple integration. 
```